cmd/notary/util.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+// getPayload is a helper function to get the content used to be verified
+// either from an existing file or STDIN.
+func getPayload(t *tufCommander) ([]byte, error) {
+
+	// Reads from the given file
+	if t.input != "" {
+		// Please be noticed that ReadFile will cut off the size if it was over 1e9.
+		// Thus, if the size of the file exceeds 1GB, the over part will not be
+		// loaded into the buffer.
+		payload, err := ioutil.ReadFile(t.input)
+		if err != nil {
+			return nil, err
+		}
+		return payload, nil
+	}
+
+	// Reads all of the data on STDIN
+	payload, err := ioutil.ReadAll(os.Stdin)
+	if err != nil {
+		return nil, fmt.Errorf("Error reading content from STDIN: %v", err)
+	}
+	return payload, nil
+}
+
+// feedback is a helper function to print the payload to a file or STDOUT or keep quiet
+// due to the value of flag "quiet" and "output".
+func feedback(t *tufCommander, payload []byte) error {
+	// We only get here when everythings goes well, since the flag "quiet" was
+	// provided, we output nothing but just return.
+	if t.quiet {
+		return nil
+	}
+
+	// Flag "quiet" was not "true", that's why we get here.
+	if t.output != "" {
+		if err := ioutil.WriteFile(t.output, payload, 0644); err != nil {
+			return err
+		}
+	}
+
+	os.Stdout.Write(payload)
```

Comment:
it seems that if we provide a value to the `output` flag, we write both to the file as well as stdout.
To be consistent with the flag description, we can remove the error-checking in the `t.output != ""` case and just `return ioutil.WriteFile(t.output, payload, 0644)`.

I'm also ok with leaving this behavior of writing to both the output file and stdout, if we update the flag description